src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/validate/PNCIdentifier.kt

```diff
@@ -5,12 +5,12 @@ import java.time.LocalDate
 
 const val LONG_PNC_ID_LENGTH = 10
 
-data class PNCIdentifier(private val inputPncId: String? = null) {
-  val inputPnc: String?
-    get() = inputPncId
+class PNCIdentifier(private val inputPncId: String? = null) {
```

Comment:
We don't seem to be using the capabilities of the data class here - we provide an alternative getter - so I have removed it